packages/styles/carbon.yml

```diff
@@ -2,7 +2,7 @@
 library:
   id: ibmdotcom-styles
   name: IBM.com Styles
-  description: IBM.com framework agnostic component styles
+  description: Apply framework-agnostic styles to any IBM.com component.
```

Comment:
is this final? airtable says in progress

Reply:
yup, Matt finalized the description